cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/ContainerCommand.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore;
+
+import io.pravega.cli.admin.CommandArgs;
+
+/**
+ * Base class for all Segment Container related commands.
+ */
+public abstract class ContainerCommand extends SegmentStoreCommand {
```

Comment:
This naming may be confusing, we already have one `ContainerCommand` in `package io.pravega.cli.admin.bookkeeper;`. Either we extract the `ContainerCommand` to a separate package and put there all the commands that currently fall into that category, or this can be just a `SegmentStoreCommand`. I don't know, it looks like something is off if we end up having `ContainerCommand` in multiple packages.

Reply:
This could be also classified as a `StorageCommand` as well, similar to the command to read data from storage that has been recently added.

Reply:
I prefer `Storage` to be commands that operate directly on LTS while `Container` to be commands that operate on containers.

At this time this will be the only command we have for `Container`, however it doesn't mean it will stay this way. We may end up finding more stuff we want to do. My recommendation is to keep it as-is (container) for now.